src/schema/controllers/user-controller.js

```diff
@@ -0,0 +1,20 @@
+import db from '../../models'
+
+export default {
```

Comment:
This notation won't be very readable when you had more resolvers. I'd suggest this notation instead:
```javascript
export const createUser = ...

export const deleteUser = ...
```